Apps/AdminWebClient/src/ClientApp/package.json

```diff
@@ -12,6 +12,7 @@
     "@fortawesome/fontawesome-svg-core": "^1.2.30",
     "@fortawesome/free-solid-svg-icons": "^5.14.0",
     "@fortawesome/vue-fontawesome": "^2.0.0",
+    "bootstrap-vue": "^2.17.3",
```

Comment:
why do we have bootstrap vue in our admin web client?

Reply:
good catch, forgot to remove it as my 1st version used the toggle button.